deltachat-ios/Controller/ChatListController.swift

```diff
@@ -528,30 +528,33 @@ class ChatListController: UITableViewController {
             }
         } else if isArchive {
             titleView.text = String.localized("chat_archived_chats_title")
-            handleMultiSelectTitle()
+            if !handleMultiSelectionTitle() {
+                navigationItem.setLeftBarButton(nil, animated: true)
+            }
         } else {
             titleView.text = DcUtils.getConnectivityString(dcContext: dcContext, connectedString: String.localized("pref_chats"))
+            if !handleMultiSelectionTitle() {
+                navigationItem.setLeftBarButton(nil, animated: true)
+                navigationItem.setRightBarButton(newButton, animated: true)
+            }
             if dcContext.getConnectivity() >= DC_CONNECTIVITY_CONNECTED {
                 titleView.accessibilityHint = "\(String.localized("connectivity_connected")): \(String.localized("a11y_connectivity_hint"))"
```

Comment:
that set the `accessibilityHint` in selection mode, i think, this needs to go to the `!if !handleMultiSelectionTitle()` branch just above.